### PR TITLE
feature: add FstUndoLogParser

### PR DESCRIPTION
**Pull request by a364176773**
### Ⅰ. Describe what this PR did
add FstUndoLogParser

### CODE REVIEW

**Comment by a364176773** [comment]
> implements java.io.Serializable class must define `private static final long serialVersionUID` field

ok

**Review by slievrly — APPROVED** [review]
LGTM

File: rm-datasource/src/main/java/io/seata/rm/datasource/undo/parser/FstSerializerFactory.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import javax.sql.rowset.serial.SerialClob;
+import io.seata.core.serializer.SerializerClassRegistry;
+import org.nustaq.serialization.FSTConfiguration;
+
+/**
+ * @author funkye
+ */
+public class FstSerializerFactory {
+
+    private static final FstSerializerFactory FACTORY = new FstSerializerFactory();
```
Comment (jsbxyyx) [inline rm-datasource/src/main/java/io/seata/rm/datasource/undo/parser/FstSerializerFactory.java:29]: The FACTORY is singleton？

**Review by wangliang181230 — APPROVED** [review]
LGTM

**Review by jsbxyyx — APPROVED** [review]
LGTM

**Pull request merged**